psamm/randomsparse.py

```diff
@@ -0,0 +1,207 @@
+# This file is part of PSAMM.
+#
+# PSAMM is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PSAMM is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PSAMM.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright 2014-2015  Jon Lund Steffensen <jon_steffensen@uri.edu>
+# Copyright 2015  Keith Dufault-Thompson <keitht547@my.uri.edu>
+# Copyright 2016  Chao Liu <lcddzyx@gmail.com>
+
+from __future__ import unicode_literals
+
+import random
+import logging
+
+from . import fluxanalysis
+from .expression import boolean
+
+from six import iteritems, string_types
+
+logger = logging.getLogger(__name__)
+
+
+class ReactionDeletionStrategy(object):
+    """Deleting reactions strategy class.
+    """
+
+    def __init__(self, model, reaction_set=None):
+        self._model = model
+        if reaction_set is None:
+            self._test_set = set(self._model.reactions)
+        else:
+            self._test_set = set(reaction_set)
+        self._reactions = set(self._test_set)
+
+    def get_total(self):
+        return self._reactions
+
```

Comment:
Maybe call this function `entities()` since we have adopted this term to refer to reactions or genes. You can even make it a property with

``` python
@property
def entities(self):
    ...
```